scripts/VermilionGym.asm

```diff
@@ -52,16 +52,16 @@ VermilionGym_ScriptPointers:
 	dw CheckFightingMapTrainers
 	dw DisplayEnemyTrainerTextAndStartBattle
 	dw EndTrainerBattle
-	dw VermilionGymScript3
+	dw VermilionGymLTSurgePostBattle
 
-VermilionGymScript3:
+VermilionGymLTSurgePostBattle:
 	ld a, [wIsInBattle]
-	cp $ff
-	jp z, VermilionGymScript_5ca8a
+	cp $ff ; did we lose?
+	jp z, VermilionGymResetScripts
 	ld a, $f0
```

Comment:
While you're in this file, if it's not too much trouble, this `ld a, $f0` can be replaced with `ld a, D_RIGHT | D_LEFT | D_UP | D_DOWN`

Reply:
Done 👍 Thanks for the tip.